src/DataBlobs/services/BlobStores.jl

```diff
@@ -170,6 +174,25 @@ function addDataBlob!(store::FolderStore{T}, entry::BlobStoreEntry, data::T) whe
     end
 end
 
+function updateDataBlob!(store::FolderStore{T},  entry::BlobStoreEntry, data::T) where T
```

Comment:
In other places update is warn then add if it exist.

Reply:
Good point, thanks!